CHANGELOG.md

```diff
@@ -110,6 +110,9 @@
 * Updated Pods and Gems dependencies.  
   [David Jennes](https://github.com/djbe)
   [#684](https://github.com/SwiftGen/SwiftGen/pull/684)
+* Use `JSONSerialization` for parsing JSONs, making `json` actions faster.  
```

Comment:
Actually we're currently having a debate about how we should call those "actions".

In the current documentation, we interchangeably call them `subcommands` (mainly that's legacy from when we didn't have a config file and those were proper command line subcommands) and `parsers`
I've started to uniformise the term we use to always be `parser` from now (see https://github.com/SwiftGen/SwiftGen/commit/aca2804a20afa43cf68f797d41b05e80cb646500 in #697) but if you have a suggestion for a better name we're all ears.

I personally think `parser` makes more sense that `action` (even from a user's perspective, as `xcassets` "action" is a thing that reads (aka parses) assets catalogs) and I fear that `action` might be too generic (like, when documenting commands like `swiftgen templates list --only <action>` or `swiftgen template cat <action> <templateName>` it feels strange in that context), wdyt?

```suggestion
* Use `JSONSerialization` instead of `Yams` for parsing JSONs, making the `json` parser faster.  
```